app/controllers/collaborators/main_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+class Collaborators::MainController < Collaborators::BaseController
+  prepend_before_action :set_collaborator, only: %i[edit update destroy]
+
+  def index
+    render_index_props
+  end
+
+  def new
+    @title = "New collaborator"
+    render inertia: "Collaborators/New", props: CollaboratorPresenter.new(seller: current_seller).new_collaborator_props
+  end
+
+  def create
+    response = Collaborator::CreateService.new(seller: current_seller, params: collaborator_params).process
+
+    if response[:success]
+      redirect_to collaborators_path, status: :see_other, notice: "Changes saved!"
+    else
+      redirect_to new_collaborator_path, inertia: { errors: { message: response[:message] } }
+    end
+  end
```

Comment:
Pair of methods working together for `/collaborators/new` page.

1. `new` loads the page
2. `create` for saving new records